lib-object-oriented/line.js

```diff
@@ -1,34 +1,77 @@
 import Tag from './tag'
 
 export default class Line {
-  constructor( content ) {
+  constructor({ content, type }) {
     this.content = content
+    this.tagType = type
+    this.symbolCount = 0
+    this.getSymbolCount()
+    this.newLinesToSpaces()
+  }
+
+  newLinesToSpaces() {
+    const tagType = this.tagType,
+          content = this.content,
+          symbolCount = this.symbolCount
+
+    const findNewLine = element => {
+      return element.type === 'new_line'
+    }
+
+    if ( tagType !== '`' || ( tagType === '`' && symbolCount === 1 ) ) {
```

Comment:
As I continued to review this block, I had to search through the entire PR in order to understand what `this.content` was - it's not clear from the variable name what the type of content is, and then its reuse in a few lines as an object is incredibly confusing... (Is it a string that is then re-used as an object?)  There's a lot of opportunity to increase the clarity and intent of this function.